adaptive/learner/sequence_learner.py

```diff
@@ -5,8 +5,8 @@
 from adaptive.learner.base_learner import BaseLearner
 
 
-class _IgnoreFirstArgument:
-    """Remove the first argument from the call signature.
+class _IndexToPoint:
```

Comment:
This name is a bit mysterious (sounds like a dict) as well as the docstring. Not sure what the better option could be.